diff -up ./pdfshuffler/pdfshuffler.py.orig ./pdfshuffler/pdfshuffler.py
--- ./pdfshuffler/pdfshuffler.py.orig	2012-01-26 14:10:15.706139193 -0600
@@ -110,7 +110,7 @@ class PdfShuffler:
             print(_("Can't load icon. Application is not installed correctly."))

         # Import the user interface file, trying different possible locations
-        ui_path = '/usr/share/pdfshuffler/pdfshuffler.ui'
         if not os.path.exists(ui_path):
             ui_path = '/usr/local/share/applications/pdfshuffler/pdfshuffler.ui'
             if not os.path.exists(ui_path):
