--- data/pdfshuffler.desktop.orig	2008-10-19 21:58:08.000000000 +0200
+++ data/pdfshuffler.desktop	2009-01-30 23:12:43.000000000 +0100
@@ -1,13 +1,10 @@
 [Desktop Entry]
 Version=1.0
-Encoding=UTF-8
 Name=Pdf-Shuffler
-Comment=PDF Merging, Rearranging and Spliting
+Comment=PDF Merging, Rearranging, and Splitting
 Type=Application
 Exec=pdfshuffler
 Icon=pdfshuffler
 Categories=Graphics;
 Terminal=false
 StartupNotify=false
-GenericName[en_GB]=
-Name[en_US]=PDF-Shuffler
